src/ResourceManager/Profile/Commands.Profile/Context/SaveAzureRMContext.cs

```diff
@@ -65,7 +67,7 @@ public override void ExecuteCmdlet()
                         throw new ArgumentException(Resources.AzureProfileMustNotBeNull);
                     }
 
-                    if (!AzureSession.DataStore.FileExists(Path) || Force ||
+                    if (!AzureSession.DataStore.FileExists(Path) || Force.IsPresent ||
```

Comment:
Verbose messages are entirely appropriate.  This is the recommended practivce for these kinds of messages in PowerShell cmdlets.

It is also a standard pattern to return nothing if successful (or throw if not) for PowerShell action cmdlets.